packages/@aws-cdk/aws-elasticsearch/test/domain.test.ts

```diff
@@ -142,12 +120,6 @@ describe('log groups', () => {
           },
           Enabled: true,
         },
-        SEARCH_SLOW_LOGS: {
```

Comment:
Need tests for these.

I think it'll be good enough if you can change these to `SEARCH_SLOW_LOGS: ABSENT` instead of removing them.

Reply:
👍